packages/validator/test/cli-validator/tests/info-and-hint.test.js

```diff
@@ -35,7 +35,7 @@ describe('test info and hint rules - OAS3', function() {
     expect(jsonOutput['errors'].length).toBe(4);
 
     // Verify warnings
-    expect(jsonOutput['warnings'].length).toBe(37);
+    expect(jsonOutput['warnings'].length).toBe(16);
```

Comment:
All of these were duplicated validations spread across different references of the same schema